README.md

```diff
@@ -1,6 +1,6 @@
 # Babble
-![image of babble landing page]() <br />
-[Babble](https://babble-lilac.vercel.app/) is a video chat web app with Zoom integration and synchronous text workspace support.
+![image of babble landing page](images/Landing.png) <br />
```

Comment:
This image is already displayed in the `Landing` section below. I don't know if it's necessary here. Up to you guys.

Reply:
Yeah, that's true. I think removing this and then having the banner image is good

Reply:
True, the banner would fit nicely 👍